### DISCUSSION TITLE
[WASM] Is there a way to import function from other modules than `env`?

### DISCUSSION THREAD

**Discussion by roblillack** [tinygo-org/tinygo#4504] [Q&A] [answered]
The documentation states this in https://tinygo.org/docs/guides/webassembly/wasm/:

```go
// This function is imported from JavaScript, as it doesn't define a body.
// You should define a function named 'add' in the WebAssembly 'env'
// module from JavaScript.
//
//export add
func add(x, y int) int
```

How would I be able to import a function from a module named differently? I tried `//export foo.bar` and `//export foo::bar` to no avail.

**Comment by deadprogram** [comment, marked as answer]
You are probably wanting the just added to `dev` branch support for `//go:wasmexport`.

Please see https://github.com/tinygo-org/tinygo/pull/4451 and https://github.com/tinygo-org/tinygo/pull/4499

**Reply by roblillack** [reply]
Thanks!

**Comment by deadprogram** [comment]
Actually this is the PR that adds what you specifically are asking for, I think https://github.com/tinygo-org/tinygo/pull/4494
